src/type-inference.md

```diff
@@ -58,7 +57,7 @@ inference works, or perhaps this blog post on
 
 [Unification in the Chalk project]: http://smallcultfollowing.com/babysteps/blog/2017/03/25/unification-in-chalk-part-1/
 
-All told, the inference context stores four kinds of inference variables as of this
+All said, the inference context stores four kinds of inference variables as of
```

Comment:
Hmm... did you intend to remove "this"?

Reply:
Yep. It makes better sense as `writing`, not `this writing`.